site/docs/hive.md

```diff
@@ -129,6 +129,15 @@ SET iceberg.catalog.glue.lock-impl=org.apache.iceberg.aws.glue.DynamoLockManager
 SET iceberg.catalog.glue.lock.table=myGlueLockTable;
 ```
 
+Register a `NessieCatalog` called `nessie`:
```

Comment:
This seems to be unrelated change, can we remove this from this PR?

Reply:
fixed